students/avalvillar/lesson4/mailroom_update.py

```diff
@@ -0,0 +1,153 @@
+"""
+    Author is Antonio V. Alvillar
+    Self-Paced-Online Python 210 UWPCE
+    December 11th 2018
+"""
+
+"""
+This function has been updated to use a dictionary and fileout.
+    There are quite a bit of prints that were used during testing. This
+    can be deleted if need be. Keeping them there in case of bug fixes
+    needed with this assignment. With the list of lists I felt it important.
+"""
+
+#Initial Donor list with donations and then print for checking.
+donors = {'Adam Alvarez': [1.25], 'Beth Bonnor':[2.73]}
+donors['Cate Campbell'] = [3.33, 4.44]
+donors['Dave Don'] = [5550.33, 1.41]
+donors['Eric Ebron'] = [12.02, 5.25, 4.25]
+#print(donors)
+
+
+#Display the Thank you note to the screen
+def display_thank_you(donor, amount): #Create an email with name an donation.
+    personal_info = {'name':donor, 'donation':amount}
+    print('\n')
+    print("{name}, \n\n    Your donation of ${donation}".format(**personal_info)
+          + " was received and we are forever grateful for your support." +
+          "\n\nSincerely,\nDonation Center\n(555) 555-5555")
+    #print("\n\n" + email)
+    print("\n\n\nReturning to the main menu.....")
+    start_up() #After email is displayed - start over.
```

Comment:
this should be handled by `while True` loop which would live in the `start_up` function, not manually